R/methods.R

```diff
@@ -134,7 +134,7 @@ split.lints <- function(x, f=NULL, ...) {
 }
 
 #' @export
-as.data.frame.lints <- function(x, row.names = NULL, optional = FALSE, ...) {
+as.data.frame.lints <- function(x, row.names = NULL, optional = FALSE, ...) { # nolint: object_name_linter. (row.names)
```

Comment:
This was actually a false negative before. `row()` is a function in `base`, but not a generic, so `row.names` is not a generic method.
Fun fact: `row.names` _is_ a generic so `row.names.x` would never lint.